File: corpus-analyzer/tests/fixtures/hallucination/sample.py
"""Sample Python file for testing chunking."""

import os
from typing import Any


class SampleClass:
    """A sample class for testing."""

    def __init__(self, value: int) -> None:
        """Initialize with a value."""
        self.value = value

    def get_value(self) -> int:
        """Return the value."""
        return self.value


def sample_function(x: int) -> int:
    """A sample function."""
    return x * 2


if __name__ == "__main__":
    obj = SampleClass(10)
    print(obj.get_value())

File: corpus-analyzer/tests/fixtures/hallucination/sample.ts
/**
 * Sample TypeScript file for testing chunking.
 */

export interface SampleInterface {
  name: string;
  value: number;
}

export class SampleClass implements SampleInterface {
  name: string;
  value: number;

  constructor(name: string, value: number) {
    this.name = name;
    this.value = value;
  }

  getInfo(): string {
    return `${this.name}: ${this.value}`;
  }
}

export function sampleFunction(x: number): number {
  return x * 2;
}

// Main execution
const obj = new SampleClass("test", 10);
console.log(obj.getInfo());
